### ISSUE TITLE
[F7-P1] Cloud chamber notebook structure and single-cycle simulation

### ISSUE THREAD

**Issue by Gorkowski** [uncscode/particula#896]
## Description

Create the foundational notebook structure for simulating cloud droplet activation and deactivation cycles in a rectangular cloud chamber. This phase establishes the notebook skeleton with imports, plotting style, chamber parameters, and implements a single activation-deactivation cycle demonstrating droplet growth to 5+ μm diameter.

## Context

Cloud chambers are essential tools for studying aerosol-cloud interactions. This notebook is part of Feature F7 (Cloud Chamber Injection Cycles Simulation) which demonstrates particula's full capabilities: particle-resolved tracking, wall loss, dilution, injection, and condensation. This first phase establishes the foundation that subsequent phases will build upon.

**Value:**
- Demonstrates particle-resolved speciated mass tracking
- Shows how seed particles activate into cloud droplets at supersaturation
- Provides beginner-friendly introduction to cloud activation physics
- Establishes patterns for subsequent phases

## Scope

**Estimated Lines of Code:** ~150 LOC (notebook cells)
**Complexity:** Medium

**Files to Create:**
- `docs/Examples/Simulations/Notebooks/Cloud_Chamber_Cycles.ipynb` (~150 LOC)

**Files to Modify:** None in this phase

### Core Implementation
- [ ] Create Jupyter notebook with proper structure and imports
- [ ] Configure matplotlib plotting style consistent with other notebooks
- [ ] Define rectangular chamber parameters (1.0m × 0.5m × 0.5m)
- [ ] Define seed species properties (ammonium sulfate with κ=0.61, sucrose with κ=0.10)
- [ ] Create particle-resolved speciated mass representation using `ParticleResolvedSpeciatedMassBuilder`
- [ ] Configure rectangular chamber wall loss via `RectangularWallLossBuilder`
- [ ] Set up κ-based activity strategy using `ActivityKappaParameterBuilder`
- [ ] Implement single activation phase at ~100.4% RH (0.4% supersaturation)
- [ ] Implement single deactivation phase at ~60-70% RH
- [ ] Demonstrate droplet growth to 5+ μm diameter at peak activation
- [ ] Visualize individual particle size trajectories through the cycle

### Beginner-Friendly Documentation
- [ ] Add markdown cells explaining cloud droplet activation physics
- [ ] Explain Köhler theory basics and κ-theory approximation
- [ ] Document each code section with clear purpose and expected output
- [ ] Include internal consistency checks (mass conservation within 1%)

### Internal Validation (no external test files needed)
- [ ] Notebook runs end-to-end without errors
- [ ] Droplets reach 5+ μm diameter at peak supersaturation
- [ ] Total mass (seed + water) conserved within 1% (accounting for wall loss)
- [ ] Set fixed random seeds for reproducibility

### Chamber Configuration

```python
import particula as par

# Rectangular cloud chamber dimensions
chamber_length = 1.0  # m
chamber_width = 0.5   # m
chamber_height = 0.5  # m
chamber_volume = chamber_length * chamber_width * chamber_height  # m³

# Build wall loss strategy using builder pattern
wall_loss_strategy = (
    par.dynamics.RectangularWallLossBuilder()
    .set_chamber_dimensions((chamber_length, chamber_width, chamber_height))
    .set_wall_eddy_diffusivity(0.001)  # m²/s
    .set_distribution_type("particle_resolved")
    .build()
)

# Wrap in runnable for execution
wall_loss = par.dynamics.WallLoss(wall_loss_strategy=wall_loss_strategy)
```

### Seed Species Properties

```python
# Ammonium sulfate (NH4)2SO4
as_molar_mass = 0.13214  # kg/mol
as_density = 1770  # kg/m³
as_kappa = 0.61  # high hygroscopicity

# Water
water_molar_mass = 0.018015  # kg/mol
water_density = 997  # kg/m³
```

### Activity Strategy Setup

```python
# Build κ-based activity strategy for ammonium sulfate
as_activity = (
    par.particles.ActivityKappaParameterBuilder()
    .set_kappa(as_kappa)
    .set_density(as_density)
    .set_molar_mass(as_molar_mass)
    .set_water_index(1)  # water is second species
    .build()
)
```

### Particle Representation Setup

```python
# Create particle-resolved speciated mass distribution
distribution_strategy = par.particles.ParticleResolvedSpeciatedMassBuilder().build()

# Build aerosol with particles of different dry diameters
# Each particle tracks [seed_mass, water_mass]
aerosol = (
    par.AerosolBuilder()
    .set_distribution_strategy(distribution_strategy)
    .set_activity_strategy(as_activity)
    # ... additional configuration
    .build()
)
```

### Key Physics to Demonstrate

1. **Size-dependent activation**: Larger particles activate at lower supersaturation (Kelvin effect)
2. **Hygroscopicity control**: κ value determines critical supersaturation
3. **Droplet growth**: Water condenses rapidly above critical supersaturation
4. **Deactivation**: Droplets shrink but don't fully evaporate at 60-70% RH

## Example Notebook Structure

```
1. Introduction and Learning Objectives
2. Imports and Plotting Style
3. Chamber Configuration
4. Seed Species Properties
5. Building the Aerosol (particle-resolved)
6. Configuring Wall Loss and Condensation
7. Running Single Activation Cycle (humid phase)
8. Running Single Deactivation Cycle (dry phase)
9. Visualizing Particle Trajectories
10. Mass Conservation Check
11. Summary and Next Steps
```

## References

**Feature Plan:**
- `adw-docs/dev-plans/features/F7-cloud-chamber-cycles-simulation.md`

**Related Documentation:**
- `adw-docs/code_style.md` - Python standards
- `adw-docs/docstring_guide.md` - Documentation format

**Existing Simulation Notebooks (for pattern reference):**
- `docs/Examples/Simulations/Notebooks/Biomass_Burning_Cloud_Interactions.ipynb`
- `docs/Examples/Simulations/Notebooks/Soot_Formation_in_Flames.ipynb`

**Core Classes:**
- `particula.particles.ParticleResolvedSpeciatedMass`
- `particula.particles.ParticleResolvedSpeciatedMassBuilder`
- `particula.dynamics.MassCondensation`
- `particula.dynamics.WallLoss`
- `particula.dynamics.RectangularWallLossStrategy`
- `particula.dynamics.RectangularWallLossBuilder`
- `particula.particles.ActivityKappaParameter`
- `particula.particles.ActivityKappaParameterBuilder`
